pygmt/__init__.py

```diff
@@ -21,8 +21,15 @@
 import atexit as _atexit
 from importlib.metadata import version
 
+from pygmt import clib
+
+# Get semantic version through setuptools-scm
+__version__ = f'v{version("pygmt")}'  # e.g. v0.1.2.dev3+g0ab3cd78
+__commit__ = __version__.split("+g")[-1] if "+g" in __version__ else ""  # 0ab3cd78
+with clib.Session() as lib:
+    __gmt_version__ = lib.info["version"]
```

Comment:
The package-level variable `__gmt_version__` is defined here.

I have to put the definition of `__gmt_version__` at the top otherwise there will be circular import errors (see https://github.com/GenericMappingTools/pygmt/actions/runs/4179854302/jobs/7240260379).